app/src/main/java/ch/sdp/vibester/activity/BuzzerScreenActivity.kt

```diff
@@ -144,13 +145,15 @@ class BuzzerScreenActivity : AppCompatActivity() {
     private fun setAnswerButton(answer: LinearLayout, button: Button, updater: BuzzerScoreUpdater, map: Map<Int, Int>) {

```

Comment:
map is buzzersToRows in this function. BuzzersToRows is global in BuzzerScreenActivity class, so it is accessible in this function without passing it as a parameter. Do you think it is better if we remove map from here as a parameter and will access buzzerToRows inside the function?